alert-database/src/main/java/com/synopsys/integration/alert/database/notification/NotificationContentRepository.java

```diff
@@ -84,4 +82,6 @@
                        + ")")
     Page<NotificationEntity> findMatchingSentNotification(@Param("searchTerm") String searchTerm, Pageable pageable);
 
+    @Query(value = "SELECT entity FROM NotificationEntity entity WHERE entity.processed = false")
+    Page<NotificationEntity> findNotProcessedNotifications(Pageable pageable);
```

Comment:
We might be able to do this without a custom query: 
https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods.query-creation

```suggestion
    Page<NotificationEntity> findByProcessedFalse(Pageable pageable);
```